frontend/src/pages/DetailedCourse/DetailedCourse.tsx

```diff
@@ -90,7 +75,12 @@ const DetailedCourse: React.FC<IProps> = ({
           <Image imageUrl={courseData.avatar} />
         </ImageWrapper>
         {isCourseApplicationSubmitted && !isCourseStatusPending && (
-          <ProgressBar size="large" text="0%" textColor="#131313" />
+          <ProgressBar
+            size="large"
+            text={isCourseCompleted ? 'Completed' : '0%'}
+            textColor="#131313"
+            variant={isCourseCompleted && 'successful'}
```

Comment:
I think it would be better to move 'Completed' and '0%' to constants.
Also maybe we should export const VARIANTS from "components/ProgressBar/ProgressBar.tsx" and reuse it instead of using strings in "variant" prop?